src/absil/ilwrite.fs

```diff
@@ -4462,105 +4474,146 @@ let writeBinaryAndReportMappings (outfile, ilg, pdbfile: string option, signer:
                  b0 reloc2; b1 reloc2; |]
           writePadding os "end of .reloc" (imageEndSectionPhysLoc - relocSectionPhysLoc - relocSectionSize)
 
-          os.Close()
-          
-          try 
-              FileSystemUtilites.setExecutablePermission outfile
```

Comment:
The call to setExecutablePermission was surrounded by a try-with - has that been maintained in the code?  Again we need to see that there is zero chance of behaviour change for on-disk file emit. thanks